graphql-jpa-query-schema/src/test/java/com/introproventures/graphql/jpa/query/schema/model/calc/CalcEntity.java

```diff
@@ -0,0 +1,42 @@
+package com.introproventures.graphql.jpa.query.schema.model.calc;
+
+import com.introproventures.graphql.jpa.query.annotation.GraphQLCalcField;
+import com.introproventures.graphql.jpa.query.annotation.GraphQLDescription;
+import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Transient;
+
+@Data
+@Entity
+public class CalcEntity {
+    @Id
+    Long id;
+
+    String title;
+
+    String info;
+
+    @Transient
+    @GraphQLCalcField
+    boolean logic = true;
+
+    @Transient
```

Comment:
Is there any reason to not try using `@Transient` annotation without introducing extra `@GraphQLCalcField` ?